validator_client/src/cli.rs

```diff
@@ -79,6 +79,19 @@ pub fn cli_app() -> Command {
                 .conflicts_with("datadir")
                 .display_order(0)
         )
+        .arg(
+            Arg::new("http-token-path")
+                .long("http-token-path")
+                .value_name("HTTP_TOKEN_PATH")
+                .help(
+                    "Path to file containing the HTTP API token for validator client authentication. \
+                    If not specified, defaults to {validators-dir}/api-token.txt. \
+                    The file will be created if it does not exist.",
+                )
+                .action(ArgAction::Set)
+                .conflicts_with("datadir")
```

Comment:
This should not conflict with `datadir`.

It should have `requires("http")` instead